cpp/modmesh/mesh/StaticMesh.hpp

```diff
@@ -88,11 +88,10 @@ struct CellType : NumberBase<int32_t, double>
 
     uint8_t id() const { return m_id; }
     uint8_t ndim() const { return m_ndim; }
-    uint8_t nnode() const { return m_nnode; }
-    uint8_t nedge() const { return m_nedge; }
-    uint8_t nsurface() const { return m_nsurface; }
-
-    uint8_t nface() const { return 2 == m_ndim ? nedge() : nsurface(); }
+    uint8_t nnode() const { return m_attrs[NNODE_IDX]; }
+    uint8_t nedge() const { return m_attrs[NEDGE_IDX]; }
+    uint8_t nsurface() const { return m_attrs[NSURFACE_IDX]; }
+    uint8_t nface() const { return m_attrs[m_ndim % 3]; } // Using mod can prevent idx out-of-boundary occurs.
```

Comment:
Please just use `m_ndim` to avoid the unnecessary mod.  `m_ndim` should never be larger than 3.

Reply:
I use `m_ndim - 1` instead of **mod**.